.github/workflows/scheduled-dependency-update.yml

```diff
@@ -0,0 +1,33 @@
+name: Scheduled Dependency Update
+
+on:
+  # run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+  # every monday at 02:00
```

Comment:
thanks for the comment 👍 server time is CET ? That usually bites me. The time doesn't matter actually as it's not holding any service. Just curious.

Reply:
It's UTC. I didn't even think of the timezone, good hint to do so 🙈